internal/shellgen/tmpl/flake_remove_nixpkgs.nix.tmpl

```diff
@@ -0,0 +1,42 @@
+{
+   description = "A devbox shell";
+
+   inputs = {
+     {{/* nixpkgs could be removed once we turn php/haskell planners into flakes */ -}}
+     nixpkgs.url = "{{ .NixpkgsInfo.URL }}";
+     {{- range .FlakeInputs }}
+     {{.Name}}.url = "{{.URLWithCaching}}";
+     {{- end }}
+   };
+
+   outputs = {
+     self,
+     nixpkgs,
+     {{- range .FlakeInputs }}
+     {{.Name}},
+     {{- end }}
+   }:
+       let
+          pkgs = nixpkgs.legacyPackages.{{ .System }};
+          {{- range $_, $pkg := .Packages }}
+          {{ $pkg.Name }}.{{ $.System }} = builtins.fetchClosure{
+             fromStore = "{{ $pkg.FromStore }}";
+             fromPath = "{{ $pkg.FromPath }}";
+          };
+          {{- end }}
```

Comment:
I think the attribute path order should be flipped to `{{ $pkg.System }}.{{ $.Name }}` to follow the flake convention. The name should also be quoted in case it has special characters.

Another option is to just inline this directly in the `builtInputs` list with something like:

```nix
[
   {{- range .Packages }}
   (builtins.fetchClosure{
      fromStore = "{{ .FromStore }}";
      fromPath = "{{ .FromPath }}";
   })
   {{- end }}
]
```